### PR TITLE
Added Dumpert provider

### PR DESCRIPTION
**Pull request by KeizerDev**
Dumpert is a dutch video platform for just funny video's. 

You can just ignore this if you do not think it's necessary to add stuff like this, but if you want to let `s` turn out to something like https://github.com/rg3/youtube-dl, you can! It's up to you  :)

### CODE REVIEW

File: providers/dumpert/dumpert.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  providers.AddProvider("dumpert", &DumpertProvider{})
+}
+
+// DumpertProvider adheres to the Provider interface.
+type DumpertProvider struct {
+}
+
+// BuildURI generates a search URL for dumpert.
+func (p *DumpertProvider) BuildURI(q string) string {
+  return fmt.Sprintf("http://www.dumpert.nl/search/ALL/%s/", url.QueryEscape(q))
```
Comment (zquestz) [inline providers/dumpert/dumpert.go]: So this doesn't seem to work for multi word searches. Also should be https.

**Comment by zquestz** [comment]
This looks just fine, for now I want to support https wherever possible. Also it seems when I do multi word searches it doesn't respect all the terms. We should make sure that works.

**Comment by KeizerDev** [comment]
@zquestz `s "dikke bmw" -p dumpert`

**Comment by KeizerDev** [comment]
@zquestz btw, it isn't actually a valid `https` site.

**Comment by zquestz** [comment]
Seems to load https for me, what makes it not valid? Seems to work on this end.

**Comment by zquestz** [comment]
Even reports in Chrome that it is using TLS 1.2

**Comment by zquestz** [comment]
We should rebase this into a single commit, and make sure it is clean to merge. Then we should be gtg.

**Comment by zquestz** [comment]
Ohh and feel free to add yourself to the Contributors in the README.

**Comment by KeizerDev** [comment]
@zquestz Done :+1:

**Comment by KeizerDev** [comment]
@zquestz Yeah, will add myself tomorrow!

**Comment by zquestz** [comment]
Last NIT, make sure to use gofmt! This doesn't have correct indentation.

**Comment by KeizerDev** [comment]
@zquestz Done!

**Comment by zquestz** [comment]
Need to rebase on master, there is a conflict and I can't merge through GH. =)

**Comment by zquestz** [comment]
Seems the facebook commit had a conflict... haha

**Comment by KeizerDev** [comment]
@zquestz Merged with master

**Pull request merged**